packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -5560,17 +5560,69 @@ describe("Teal Opcodes", function () {
       );
     });
 
+    it("uncover: n = 1", () => {
+      push(stack, 4); // stack state = [1, 2, 3, 4]
+      const op = new Uncover(['1'], 1);
```

Comment:
```suggestion
      const op = new Uncover([1n], 1);
```
I think we can use `1n` (big number notation) instead of string. Could you check please?

Reply:
No, We can't. Constructor of opcode is Opcode(args: string[], line: number) so args is string[] we can't use BigInt.

Reply:
ah, it will be part of parsing. Thanks.